packages/tradershub/src/components/TradersHubContent/TradersHubContent.tsx

```diff
@@ -1,24 +1,16 @@
-import React, { Fragment } from 'react';
+import React from 'react';
+import { twMerge } from 'tailwind-merge';
 import { CFDSection, OptionsAndMultipliersSection } from '@/components';
 import { useRegulationFlags } from '@/hooks';
 
 const TradersHubContent = () => {
     const { isEU } = useRegulationFlags();
 
-    if (isEU) {
```

Comment:
Instead of returning twice, used css to do the conditional rendering and just returned once